pydicom/tests/test_coding.py

```diff
@@ -2,36 +2,35 @@
 
 import pytest
 
-from pydicom.coding import Code
+from pydicom.sr.coding import Code
 from pydicom.uid import UID
 
 
 class TestCode(unittest.TestCase):
```

Comment:
Should remove the reference to `unittest` at some point.

Reply:
Yeah, let's leave and do cleanup as with parameterizing...